jsk_pcl_ros/src/cluster_point_indices_decomposer_nodelet.cpp

```diff
@@ -502,8 +502,33 @@ namespace jsk_pcl_ros
       }
     }
     else {
-      segmented_cloud_transformed = segmented_cloud;
-      is_center_valid = pcl::compute3DCentroid(*segmented_cloud_transformed, center) != 0;
+      if (use_pca_) {
+        if (segmented_cloud->points.size() >= 3) {
+          Eigen::Vector4f pca_centroid;
+          pcl::compute3DCentroid(*segmented_cloud, pca_centroid);
+          Eigen::Matrix3f covariance;
+          pcl::computeCovarianceMatrixNormalized(*segmented_cloud, pca_centroid, covariance);
+          Eigen::SelfAdjointEigenSolver<Eigen::Matrix3f> eigen_solver(covariance, Eigen::ComputeEigenvectors);
+          Eigen::Matrix3f eigen_vectors_pca = eigen_solver.eigenvectors();
+          /// This line is necessary for proper orientation in some cases. The numbers come out the same without it, but
```

Comment:
Comments can start with `//`. Is there any special reason to use `///`?

And it seems the comment does not end for now. Please update it.

Reply:
Also I fixed comment